proto/remote_execution.proto

```diff
@@ -2239,6 +2242,15 @@ message UsageStats {
 
   // IO PSI metrics.
   PSI io_pressure = 7;
+
+  message Timeseries {
+    // Duration between each sample.
+    google.protobuf.Duration period = 1;
+    
+    // Observed samples.
+    // TODO: maybe do delta-compression and/or RLE to reduce data size.
+    repeated int32 samples = 2;
```

Comment:
this could be a uint32, unless there's some metric that could be negative.

Reply:
I think we are following some best practice about using signed ints because it makes it less likely to run into bugs due to overflow when doing math on the values.